herbert_core/utilities/general/is_folder.m

```diff
@@ -0,0 +1,26 @@
+function ok = is_folder(name)
+% tests if name is a valid folder
```

Comment:
It would help to include a note here that it calls `isfolder` in MATLAB >= 2016b